media-gfx/yafaray/yafaray-3.3.0-r2.ebuild

```diff
@@ -80,7 +80,8 @@ src_install() {
 
 	python_domodule "${BUILD_DIR}/src/bindings/yafaray_v3_interface.py"
 	python_domodule "${BUILD_DIR}/src/bindings/_yafaray_v3_interface.so"
-	rm -v "${D}"usr/$(get_libdir)/{yafaray_v3_interface.py,_yafaray_v3_interface.so} || die
+	rm -v "${ED}"/usr/$(get_libdir)/{yafaray_v3_interface.py,_yafaray_v3_interface.so} || die
+	rm -rfv "${ED}"/usr/share/doc/${PN} || die
```

Comment:
why do you need the force here?

Reply:
You're right, it's not needed. I probably was thinking it's needed because of the recursive option.